ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -701,6 +701,16 @@ public void shouldRecoverDropWithTerminate() {
     shouldRecover(commands);
   }
 
+  @Test
+  public void shouldRecoverIfNotExists() {
```

Comment:
I don't think this test will pass because the restore issue is not fixed yet. The PR is only to prevent writing the CREATE IF NOT EXISTS in the command topic, isn't it?